### PR TITLE
Added fluid property dictionaries and implemented Gerry's algorithm …

### PR DESCRIPTION
**Pull request by cognar**
…for pipe sizing

### CODE REVIEW

File: code/PipeFlowMain.py [hunk 1]
```diff
@@ -64,3 +97,17 @@ def pipeSize(L, Q, hl, e):
...
+    pipeDiameter1 = 0.5  # Roughly the midpoint of schedule 40 pipe sizes
+    while True:
+        V = (4 * Q) / (math.pi * pipeDiameter1 ** 2)
+        f = getFrictionFactor(V, pipeDiameter1, e)
+        pipeDiameter2 = ((8*L*Q**2*f)/(math.pi**2*g*hl))**0.2
+        if pipeDiameter2 == pipeDiameter1:
+            return pipeDiameter2
+        else:
+            pipeDiameter1 = pipeDiameter2
+
```
Comment (smohler) [inline code/PipeFlowMain.py:113]: Could you rewrite to ensure this function terminates. Two floats will never be identically equal. So change it to (D_old -D_new)^2<epsilon (small number).

**Review by smohler — COMMENTED** [review]
See minor fix for the exactPipeSize function

File: code/PipeFlowMain.py (hunk 1 above)
Comment (cognar) [reply code/PipeFlowMain.py:113]: I changed the equality to use the math.isclose method, Thanks for the heads up, I had forgotten about floating point errors

**Review by smohler — APPROVED** [review]
Good job Connor

**Pull request merged**